src/libtest/lib.rs

```diff
@@ -1720,7 +1710,7 @@ mod tests {
                 should_panic: ShouldPanic::No,
                 allow_fail: false,
             },
-            testfn: DynTestFn(Box::new(move |()| f())),
+            testfn: DynTestFn(Box::new(move || f())),
```

Comment:
These can simply be `Box::new(f)` I think.

Reply:
addressed in the latest force push